internal/scout/advise/k8s.go

```diff
@@ -83,15 +90,39 @@ func Advise(ctx context.Context, cfg *scout.Config, pod v1.Pod) error {
 			advice = append(advice, storageAdvice)
 		}
 
-		fmt.Println(scout.EmojiFingerPointRight, pod.Name)
-		for _, msg := range advice {
-			fmt.Println(msg)
+		if cfg.Output != "" {
+			outputToFile(ctx, cfg, pod, advice)
+		} else {
+			for _, msg := range advice {
+				fmt.Println(msg)
+			}
 		}
 	}
 
 	return nil
 }
 
+// outputToFile writes resource allocation advice for a Kubernetes pod to a file.
+func outputToFile(ctx context.Context, cfg *scout.Config, pod v1.Pod, advice []string) error {
+	file, err := os.OpenFile(cfg.Output, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return errors.Wrap(err, "failed to open file")
+	}
+	defer file.Close()
+
+	if _, err := file.WriteString(fmt.Sprintf("- %s\n", pod.Name)); err != nil {
+		return errors.Wrap(err, "failed to write pod name to file")
+	}
+
+	for _, msg := range advice {
+		if _, err := file.WriteString(fmt.Sprintf("%s\n", msg)); err != nil {
```

Comment:
```suggestion
if _, err := fmt.Fprintf(file, "%s\n", msg); err != nil {
```

Reply:
No disagreements here, your approach seems cleaner, but would you mind explaining why you prefer fmt.Fprintf to file.WriteString?